torch/_inductor/lowering.py

```diff
@@ -2087,7 +2087,6 @@ def is_aligned(x):
 make_fallback(aten._thnn_fused_lstm_cell, require_dense)
 make_fallback(aten.topk)
 make_fallback(aten.upsample_bicubic2d_backward, require_contiguous)
-make_fallback(aten._scaled_mm.default, constrain_to_fx_strides)
```

Comment:
You are not lowering `_scaled_mm` in this PR, right? Removing `_scaled_mm` here will just make it use a silent implicit fallback.

Reply:
ah yeah I wasn't sure about this (@ipiszy was asking about it too above). I'll put it back